annual_meantemp.json

```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1",
+    "title": "Regional average climate observations UK Annual Mean Temperature",
+    "summary": "Climate observations that have been averaged across UK geographic areas",
+    "description": "Derived from https://catalogue.ceda.ac.uk/uuid/4dc8450d889a491ebb20e724debe2dfb",
+    "dataset_version": "v1.1.0.0",
+    "publisher": "https://www.gov.uk/government/organisations/met-office",
+    "creator": "https://www.gov.uk/government/organisations/met-office",
+    "themes": "http://gss-data.org.uk/def/gdp#climate-change",
+    "license": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+    "keywords": [
+        "climate",
+        "temperature"
+    ],
+    "columns": {
+        "Year": {
+            "from_template": "year"
+        },
+        "Geography": {
+            "label": "Met Office Geography",
```

Comment:
Do these directly map to the administrative geographies of England, Wales, Scotland, and Northern Ireland? If so, I suggest you use the ONS Geography codes.

E09000001, etc.

Reply:
We can map the Geography names to the ONS country codes in the CSVs if that makes it easier for you to decode.

I.e.

UK: K02000001
England: E92000001
Wales: W92000004
Scotland: S92000003
Northern Ireland: N92000002

If we do this, do we also need to update the Geography from_existing link?